ref/feature/openidConnectServer/examples.adoc

```diff
@@ -1,5 +1,6 @@
-The OpenID Connect Provider(OP) feature can be configured to manage user authentication for OpenID Connect(OIDC) clients and microservices during development.
+== Examples
 
+The OpenID Connect Provider(OP) feature can be configured to manage user authentication for OpenID Connect(OIDC) clients and microservices during development.
```

Comment:
I think there needs to be a space between `...Provider` and `(OP)`. Same thing for `...Connect` and `(OIDC)`.